lms/envs/certificates_html_view.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
I'm not sure if we've defined how we would support multiple langauges in certificates